autoload/multiple_cursors.vim

```diff
@@ -1179,6 +1179,10 @@ function! s:end_latency_measure()
   let s:skip_latency_measure = 0
 endfunction
 
+function! s:get_time_in_ms()
+  return substitute(reltimestr(reltime()), '\.\(...\).*', '\1', '')
```

Comment:
the regex here is weird, shouldn't this return a number?

Reply:
Thanks for the catch. Fixed!